### PR TITLE
Reorganize and modify package info

### PR DESCRIPTION
**Pull request by smusali**
- add `CHANGELOG.md`
- update `LICENSE.md`
- add `.circleci/config.yml`
- update `Gruntfile.js`
- update building and packaging for `Darwin`, `Ubuntu`, and `RedHat`
- update `.nuspec`
- fix lint issues

### CODE REVIEW

**Comment by smusali** [comment]
@ydshah2, ready for `Linux` and `Mac` and gonna work on `Windows` one tomorrow morning

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [v2.0.0] - 2020-05-14
+### Fixed
+- Fix `Cannot read property 'statusCode' of undefined` when updating [#40](https://github.com/logdna/logdna-cli/pull/40)
+
+### Changed
+- Write information output to stderr for tail and seaarch [#43](https://github.com/logdna/logdna-cli/pull/43)
```
Comment (ydshah2) [inline CHANGELOG.md:12]: typo 'search'. +1 otherwise 👍

Comment (smusali) [reply CHANGELOG.md]: Done!

**Comment by smusali** [comment]
@ydshah2, added newer things - it's ready to get final review

**Pull request merged**